node/core/backing/src/lib.rs

```diff
@@ -883,6 +890,9 @@ impl util::JobTrait for CandidateBackingJob {
 				}
 			};
 
+			drop(_span);
+			drop(span);
```

Comment:
Why drop the outer span early here?

Reply:
The remaining ones are either blocking, or waiting duration, but you are correct that it should span the whole block and have two more subspans.